src/main/java/gov/nih/ncats/molwitch/spi/ChemicalImpl.java

```diff
@@ -228,6 +229,8 @@ public String toString() {
 	void expandSGroups();
 	void generateCoordinates() throws MolwitchException;
 	void flipChirality(Stereocenter s);
-	
-	
+
+	default Class getFactoryClass() {
```

Comment:
We shouldn't need this. If we're unsure what the factory class is, we can always just do:

```
ChemicalImplFactory defFAC = ImplUtil.getChemicalImplFactory();
```

The Chemical object itself shouldn't need to know this.

That said, we COULD have this method still and just have it do the logic above, and return the actual ChemicalImplFactory ... I could go either way on this.